src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -220,32 +222,48 @@ public async Task<Manifests> UpdateManifestsAutonomously(Manifests manifests)
                 this.InstallerUrls = installerManifest.Installers.Select(i => i.InstallerUrl).Distinct().ToArray();
             }
 
+            // Generate list of InstallerUpdate objects and parse out any specified architecture overrides.
+            List<InstallerUpdateHelper> installerUpdateHelpers = this.ParseInstallerUrlsForArchOverride(this.InstallerUrls.ToList());
+
+            // If the installer update list is null there was an issue when parsing for architecture override.
+            if (installerUpdateHelpers == null)
+            {
+                return null;
+            }
+
+            // Reassign list with parsed installer urls.
+            this.InstallerUrls = installerUpdateHelpers.Select(x => x.InstallerUrl);

```

Comment:
If you keep this, use .ToList() or .ToArray(). If it's just the IEnumerable, then it'll have to be re-evaluated on every access

Reply:
Added .ToList() as per suggestion since I'm deciding to keep this line.